cmd/account/delegate.go

```diff
@@ -59,14 +62,27 @@ var delegateCmd = &cobra.Command{
 			cobra.CheckErr(err)
 		default:
 			// ParaTime delegation.
-			cobra.CheckErr("delegations within ParaTimes are not supported; use --no-paratime")
+			// TODO: This should actually query the ParaTime (or config) to check what the consensus
```

Comment:
It would be nice to have this actually implemented since the CLI is kind of a reference implementation and the web wallet team could use the same approach.